cpu/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(SOURCES
     preconditioner/block_jacobi_kernels.cpp
     solver/bicgstab_kernels.cpp
     solver/fcg_kernels.cpp
+    solver/cgs_kernels.cpp
```

Comment:
minor: the order in this file is very weird, technically I think we wanted everything, whether for header files or CMake declarations to be alphabetically sorted.